src/HostProgram.elm

```diff
@@ -1,34 +1,35 @@
-module HostProgram exposing (create, Model, Msg)
+module HostProgram exposing (Model, Msg, create)
 
 {-| The FrameRouter module is the Elm code that backs the frame-router custom element
 in the iframe-coordinator toolkit. It handles mapping URL routes to clients displayed
 in a child frame as well as message validation and routing within the parent application.
 
 This module is not currently designed for stand-alone use. You should instead use the
-custom elements defined in LINK_TO_JS_LIB to create seamless iframe applications
+custom elements defined in LINK\_TO\_JS\_LIB to create seamless iframe applications
```

Comment:
This is elm-format or vscode doing this.  Can clean up manually.

Reply:
it's elm-format. Don't worry about it, this stuff will get changed later anyway.